kodilla-testing2/src/main/java/com/kodilla/testing2/facebook/FacebookTestingApp.java

```diff
@@ -0,0 +1,32 @@
+package com.kodilla.testing2.facebook;
+
+import com.kodilla.testing2.config.WebDriverConfig;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.Select;
+
+public class FacebookTestingApp {
+
+    public static final String XPATH_SELECT_DAY = "//div[contains(@id, \"registration_container\")]/div/div/div/div/form/div/div/div/span/span/select[1]";
```

Comment:
Let assume that Facebook introduces small changes in the layout of the registration form - adds one div. Then your solution has to be rewritten. Make it more robust, and use information that "select" elements have meaningful names: day, month, year.